### DISCUSSION TITLE
Stable Diffusion XL

### DISCUSSION THREAD

**Discussion by AltayYuzeir** [SupaGruen/SD-CheatSheet#20] [General]
Hello. Thank you for this comprehensive style guide, not only in the area of generative AI, but also in art in general.
I am aware SD 1.5 is still extremely popular for all kinds of fine-tuning and modifications and as such SD 1.5 is still widely used.
I am wondering if there are any plans to re-make this projects also for SD XL?

**Comment by KJamesDyland** [comment]
SDXL is better in almost everything than 1.5 and I say almost because the image quality is not as good in SDXL as in 1.5, which for now beats it, but it will gradually get closer to such a good quality that there will be no difference, but it will take time

**Comment by SupaGruen** [comment]
> Hello. Thank you for this comprehensive style guide, not only in the area of generative AI, but also in art in general. I am aware SD 1.5 is still extremely popular for all kinds of fine-tuning and modifications and as such SD 1.5 is still widely used. I am wondering if there are any plans to re-make this projects also for SD XL?

Hi Altay,

Thanks for your kind words. Generating all the images on my small 3060 was a lotta work, but I probably won't do that for SDXL or any upcoming models (SD3?). I'm still interested, but I currently just don't have the time (it would take much longer) and it feels like such an uncertain situation with StableAi right now.

The code—HTML and JavaScript—can be easily adapted, so who knows?

All the best to you!
Jo
